cczoo/horizontal_fl/hfl-tensorflow/Makefile

```diff
@@ -16,9 +16,8 @@
 ################################# CONSTANTS ###################################
 
 GRAMINEDIR ?= ../..
-SGX_SIGNER_KEY ?= $(GRAMINEDIR)/Pal/src/host/Linux-SGX/signer/enclave-key.pem
-
-include $(GRAMINEDIR)/Scripts/Makefile.configs
+GRAMINE_PKGLIBDIR ?= /usr/lib/x86_64-linux-gnu/gramine
```

Comment:
Can we move the two environment variables (GRAMINE_PKGLIBDIR  and ARCH_LIBDIR) into the docker file? Then we can keep the Makefile OS independently.

Reply:
> Can we move the two environment variables (GRAMINE_PKGLIBDIR and ARCH_LIBDIR) into the docker file? Then we can keep the Makefile OS independently.

I think it's a good solution to fix the packages not found problems that are caused by gramine.